scripts/utils/sign_and_send.js

```diff
@@ -45,7 +60,7 @@ module.exports = function (web3 = web3, artifacts = artifacts) {
       ADDRESS_0,
       nonce
     )
-    const privateKey = process.env.PK
+    const privateKey = "0x" + process.env.PK
```

Comment:
Could you implement here your proposed solution from the chat, so that no matter how the user is providing the PK?

Reply:
This is just a placeholder to make the testplan work for the time being. I can do the proposed change in a separate PR.

Reply:
https://github.com/gnosis/dex-liquidity-provision/pull/212

Reply:
Prima!